src/olympia/abuse/models.py

```diff
@@ -235,13 +235,35 @@ def notify_reporters(self, action_helper):
         )
 
     def handle_job_recreated(self, new_job_id):
-        new_job, _ = CinderJob.objects.update_or_create(
+        from olympia.reviewers.models import NeedsHumanReview
+
+        new_job, created = CinderJob.objects.update_or_create(
             job_id=new_job_id,
             defaults={
                 'resolvable_in_reviewer_tools': True,
                 'target_addon': self.target_addon,
             },
         )
+        # If this forward has been combined with an existing non-forwarded job we need
+        # to clear the duplicate NHR
```

Comment:
Could you clarify that comment for me and future readers of that code? I suppose you mean that the duplicate NHR has a different `reason` (`ABUSE_ADDON_VIOLATION` vs `CINDER_ESCALATION`), which is why we can safely clear the old one?

Reply:
https://github.com/mozilla/addons-server/pull/22683/commits/6657358f084879812247fe4acb07b9ba6958b56a